packages/e2e-contract-tests/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
```

Comment:
Also we have a root ignore file on this one we should add only the project specific files